### PR TITLE
grid_printer assignment

### PR DESCRIPTION
**Pull request by jakedaylong**
grid_print.py ready for evaluation for assignment

### CODE REVIEW

**Comment by jakedaylong** [comment]
Added FizzBuzz assignment

**Comment by jakedaylong** [comment]
Added series.py assignment refactored fibonacci, lucas, and series_sum.

**Comment by natasha-aleksandrova** [comment]
@jakedaylong your PR contains 52 files and most of them are configs. Can you please try to clean them up?

1. make sure your git status is clean, commit and push any untracked files that you've edited.
2. add a .gitignore file to your directory -> students/jacob_daylong/.gitignore` ; inside of that file add this `.spyproject*`
3. run below command to clear your git history
`git rm -r --cached . && git add students/jacob_daylong && git commit -am "Remove ignored files" && git push`

**Comment by jakedaylong** [comment]
Corrected, removed spyproject files

**Comment by natasha-aleksandrova** [comment]
thanks!

File: students/jacob_daylong/Lesson 2/Fibonacci_Series/series.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    Check built in for fibonacci, default with only 1 argument, Lucas requires arguments.
+    """
+
+    if n==0:
+        return n0
+    elif n==1:
+        return n1
+    else:
+        return sum_series(n-1, n0, n1)+sum_series(n-2, n0, n1)
+    pass
```
Comment (natasha-aleksandrova) [inline students/jacob_daylong/Lesson 2/Fibonacci_Series/series.py:22]: want to make sure to remove these `pass` statements to keep the code clean

File: students/jacob_daylong/Lesson 3/slicing_lab/slicing_lab.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+print()
+print("Reverse")
+print(reversal(txt))
+print(reversal(numbers))
+print()
+
+def thirds(seq):
+    """Rearrange thirds of the element"""
+    seq = seq[9:] + seq[0:4] + seq[5:8]
```
Comment (natasha-aleksandrova) [inline students/jacob_daylong/Lesson 3/slicing_lab/slicing_lab.py:70]: this should be dynamic to work with various lengths.

File: students/jacob_daylong/Lesson 3/slicing_lab/slicing_lab.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+print()
+print("Remove every other")
+print(every_other(txt))
+print(every_other(numbers))
+print()
+
+def first_four_last_four(seq):
+    """Remove first four and last four elements"""
+    seq = seq[4:-4]
+    return seq
```
Comment (natasha-aleksandrova) [inline students/jacob_daylong/Lesson 3/slicing_lab/slicing_lab.py:49]: the instructions ask to remove first four, last four and every other element

File: students/jacob_daylong/Lesson 3/slicing_lab/slicing_lab.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+print()
+print("Exchange first and last, numbers")
+print(exchange_first_last(txt))
+print(exchange_first_last(numbers))
+print()
+
+def every_other(seq):
+    """Remove every other element"""
+    seq = seq[1::2]
```
Comment (natasha-aleksandrova) [inline students/jacob_daylong/Lesson 3/slicing_lab/slicing_lab.py:37]: the <start> in your slice syntax should be empty (zero)

**Review by natasha-aleksandrova — COMMENTED** [review]
@jakedaylong nice work overall - few comments to address in the slice exercise.

**Pull request merged**